release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3311,23 +3311,76 @@ This section will continue to be updated over time to provide notes on enhanceme
+{product-title} release {product-version}.2, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4316[RHSA-2024:4316] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:4319[RHBA-2024:4319] advisory.
 
 Space precluded documenting all of the container images for this release in the advisory.
 
 You can view the container images in this release by running the following command:
 
 [source,terminal]
 ----
-$ oc adm release info 4.16.0 --pullspecs
+$ oc adm release info 4.16.2 --pullspecs
 ----
-//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+[id="ocp-4-16-2-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, for clusters upgraded from older versions of {product-title}, enabling `kdump` on an OVN-enabled cluster sometimes prevented the node from rejoining the cluster or returning to the `Ready` state.
+With this release, stale data is removed from older {product-title} versions and ensures this stale data is always cleaned up. The node can now start correctly and rejoin the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-36198[*OCPBUGS-36198*])
+
+* Previously, unexpected output would appear in the terminal when creating an installer-provisioned infrastructure (IPI) cluster. With this release, the issue has been resolved and the unexpected output no longer appears. (link:https://issues.redhat.com/browse/OCPBUGS-36156[*OCPBUGS-36156*])
+
+* Previously, the {product-title} console did not show filesystem metrics on the nodes list. With this release, the filesystem metrics now appear in the nodes table. (link:https://issues.redhat.com/browse/OCPBUGS-35946[*OCPBUGS-35946*])
+
+* Previously, the Prometheus dashboard showed up empty for non-multi-cluster environments. With this release, the dashboard populates the dashboard panels as expected for both cases. (link:https://issues.redhat.com/browse/OCPBUGS-35904[*OCPBUGS-35904*])
+
+* Previously, a regression in 4.16.0 caused new baremetal installer-provisioned infrastructure (IPI) installations to fail when proxies were used. This was caused by one of the services in the bootstrap virtual machine (VM) trying to access IP address 0.0.0.0 through the proxy. With this release, this service no longer accesses 0.0.0.0. (link:https://issues.redhat.com/browse/OCPBUGS-35818[*OCPBUGS-35818*])
+
+* Previously, the current IBM Power Virtual Server (Power VS) Cluster API provider waited for some resources to be created before creating the load balancers, which sometimes resulted in the load balancers not being created before the 15 minute timeout. With this release, the timeout has been increased. (link:https://issues.redhat.com/browse/OCPBUGS-35722[*OCPBUGS-35722*])
```

Comment:
`{cap-ibm-first}` resolves to "Cluster API Provider IBM Cloud", which is a different name but apparently they are using [the same Cluster API Provider](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud) for Power VS. Added `{ibm-power-server-title}` ("IBM Power Virtual Server") to try to keep it clear that Power VS was impacted as per original text.

```suggestion
* Previously, the {cap-ibm-first} waited for some resources to be created before creating the load balancers on {ibm-power-server-title} clusters. This delay sometimes resulted in the load balancers not being created before the 15 minute timeout. With this release, the timeout has been increased. (link:https://issues.redhat.com/browse/OCPBUGS-35722[*OCPBUGS-35722*])
```